src/test/regress/greenplum_schedule

```diff
@@ -177,6 +177,7 @@ test: aggregate_with_groupingsets
 
 test: nested_case_null sort bb_mpph
 test: bb_memory_quota memconsumption
+test: DML_partition_pruning
```

Comment:
All test names are in lower case. I think this test name should be in lower case too.

Reply:
There is at least one opposite example: `src/test/regress/sql/DML_over_joins.sql`